ddtrace/contrib/anthropic/utils.py

```diff
@@ -0,0 +1,284 @@
+import sys
+from typing import AsyncGenerator
+from typing import Dict
+from typing import Generator
+from typing import Tuple
+
+from ddtrace.internal.logger import get_logger
+from ddtrace.internal.utils import get_argument_value
+from ddtrace.vendor import wrapt
+
+
+log = get_logger(__name__)
+
+
+def handle_stream_response(integration, resp, args, kwargs, span):
+    if _is_async_generator(resp):
+        return TracedOpenAIAsyncStream(resp, integration, span, args, kwargs)
+    elif _is_generator(resp):
+        return TracedOpenAIStream(resp, integration, span, args, kwargs)
+    elif _is_stream_manager(resp):
+        return TracedOpenAIStreamManager(resp, integration, span, args, kwargs)
```

Comment:
The names of the Traced stream classes might need to be changed with `Anthropic` instead of `OpenAI`